### PR TITLE
CAMEL-18601: Add assertMockEndpointsSatisfied to junit5 CamelTestSupport

### PR DESCRIPTION
**Pull request by mcarlett**

### CODE REVIEW

File: components/camel-test/camel-test-junit5/src/main/java/org/apache/camel/test/junit5/CamelTestSupport.java [hunk 1]
```diff
@@ -982,6 +983,27 @@ protected Language assertResolveLanguage(String languageName) {
...
+        MockEndpoint.assertIsSatisfied(context);
+    }
+
+    /**
+     * Asserts that all the expectations of the Mock endpoints are valid
+     *
+     * @see           MockEndpoint#assertIsSatisfied(CamelContext, long, TimeUnit)
+     *
+     * @param timeout
+     * @param unit
```
Comment (essobedo) [inline components/camel-test/camel-test-junit5/src/main/java/org/apache/camel/test/junit5/CamelTestSupport.java:1001]: Please either document it or remove it

**Review by essobedo — COMMENTED** [review]
Hi, thx for the contribution. Just one small remark

**Review by orpiske — CHANGES_REQUESTED** [review]
The method `assertIsSatisfied` was removed from CamelTestSupport as part of a preparation to cleanup this class (please see https://github.com/apache/camel/pull/8463). This class is way too complex and - if possible - I'd like to start cleaning up.

**Comment by mcarlett** [comment]
ok @orpiske , thanks for the clarification and sorry, I missed this PR, I can close it and I will update the documentation as well [test-junit5](https://camel.apache.org/components/3.18.x/others/test-junit5.html#_simple_unit_test_example)

**Comment by orpiske** [comment]
> ok @orpiske , thanks for the clarification and sorry, I missed this PR, I can close it and I will update the documentation as well [test-junit5](https://camel.apache.org/components/3.18.x/others/test-junit5.html#_simple_unit_test_example)

Thank you! Much appreciated for you looking at that - I almost forgot about it.

**Pull request closed without merging**